boards/posix/nrf52_bsim/CMakeLists.txt

```diff
@@ -13,57 +13,11 @@ if (NOT DEFINED ENV{BSIM_OUT_PATH})
  https://babblesim.github.io/folder_structure_and_env.html")
 endif()
 
-#Let's check that the HW models are the needed version or newer
-#That is, that the needed tag is somewhere in the past of its branch
-if(NOT DEFINED ENV{NO_NRF52_BSIM_VERSION_WARNING})
-  if(GIT_FOUND) #boilerplate.cmake searches for git
-    file(STRINGS "hw_models_version" NRF52_HW_MODELS_TAG) #desired version
-    execute_process(COMMAND ${GIT_EXECUTABLE} merge-base --is-ancestor
-      ${NRF52_HW_MODELS_TAG} HEAD
-      WORKING_DIRECTORY $ENV{BSIM_COMPONENTS_PATH}/ext_NRF52_hw_models/
-      OUTPUT_VARIABLE NRF52_HW_MODELS_TAG_FOUND
-      OUTPUT_STRIP_TRAILING_WHITESPACE
-      ERROR_STRIP_TRAILING_WHITESPACE
-      ERROR_VARIABLE stderr
-      RESULT_VARIABLE return_code
-    )
-    if(return_code)
-      message(WARNING "The NRF52 HW models are out of date\
- (${NRF52_HW_MODELS_TAG} needed at least); Please update them or expect\
- problems!\nReported error while trying to find the tag: \"${stderr}\"")
-
-      #let's print the latest actual tag for the available models:
-      execute_process(COMMAND ${GIT_EXECUTABLE} describe --tags
-        WORKING_DIRECTORY $ENV{BSIM_COMPONENTS_PATH}/ext_NRF52_hw_models/
-        OUTPUT_VARIABLE NRF52_HW_MODELS_TAG_FOUND
-        OUTPUT_STRIP_TRAILING_WHITESPACE
-        ERROR_STRIP_TRAILING_WHITESPACE
-        ERROR_VARIABLE stderr
-        RESULT_VARIABLE return_code
-      )
-      message(STATUS "Found NRF52 models version ${NRF52_HW_MODELS_TAG_FOUND}")
-
-      message(FATAL_ERROR "To disable this check set the environment variable\
- NO_NRF52_BSIM_VERSION_WARNING")
-
-    elseif(CMAKE_VERBOSE_MAKEFILE)
-      message(STATUS "nrf52_bsim: git merge-base --is-ancestor\
- ${NRF52_HW_MODELS_TAG} HEAD,
- stdout: ${NRF52_HW_MODELS_TAG_FOUND}
- stderr: ${stderr}")
-    endif()
-  endif()
-endif()
-
 zephyr_library()
 zephyr_library_compile_definitions(NO_POSIX_CHEATS)
 
 zephyr_include_directories(
```

Comment:
Still needed?

Reply:
Thanks.
After a quick look it would seem it is not anymore. So it should be possible to remove it in a further PR (I would not mix it with this PR though)

Reply:
Why not? It seems like dead code that's confusing and only needed due to lines that are being removed by this PR.

Reply:
The `fake` directory is still there, so I would also remove it.
It is just an unrelated change to this PR. So I would prefer to keep it in a separate commit.

Reply:
Ah, sorry for noise. I thought you were just putting something in there to keep the list nonempty.